spark-connector/spark-common/src/main/java/com/datastrato/gravitino/spark/connector/plugin/GravitinoDriverPlugin.java

```diff
@@ -36,14 +38,17 @@ public class GravitinoDriverPlugin implements DriverPlugin {
 
   private static final Logger LOG = LoggerFactory.getLogger(GravitinoDriverPlugin.class);
 
-  private GravitinoCatalogManager catalogManager;
-  private List<String> gravitinoDriverExtensions = new ArrayList<>();
-  private boolean enableIcebergSupport = false;
-
   @VisibleForTesting
   static final String ICEBERG_SPARK_EXTENSIONS =
       "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions";
 
+  private GravitinoCatalogManager catalogManager;
+  private final List<String> gravitinoIcebergExtensions =
+      Arrays.asList(
+          GravitinoIcebergSparkSessionExtensions.class.getName(), ICEBERG_SPARK_EXTENSIONS);
```

Comment:
Yes, i tested locally.
![image](https://github.com/datastrato/gravitino/assets/94670132/0e045eec-0a7c-4b34-9edb-57f56beae13f)

Reply:
cc @FANNG1

Reply:
I don't want to use `com.datastrato.gravitino.spark.connector.iceberg.extensions.GravitinoIcebergSparkSessionExtensions`, and if changing the package path when refactoring, I'll also need to change this.

Reply:
I just afraid it throws class not found exception if using `GravitinoIcebergSparkSessionExtensions` directly. since it works without iceberg runtime jar, it's ok for me